packages/Combobox/README.md

```diff
@@ -0,0 +1,51 @@
+# Combobox
+
+A combobox has all the features a select does as well as filtering and clearing results.
+
+<Example is="custom" />
```

Comment:
The close button should not be displayed when no option is selected 

<img width="335" alt="Capture d’écran, le 2023-02-13 à 20 22 54" src="https://user-images.githubusercontent.com/2379339/218613828-b8f9971f-a9b5-495e-b7eb-228bcb85fcf6.png">